src/messaging/ReliableMessageContext.cpp

```diff
@@ -259,8 +259,15 @@ CHIP_ERROR ReliableMessageContext::SendStandaloneAckMessage()
     // Send the null message
     if (mExchange != nullptr)
     {
-        err = mExchange->SendMessage(Protocols::SecureChannel::MsgType::StandaloneAck, std::move(msgBuf),
-                                     BitFlags<SendMessageFlags>{ SendMessageFlags::kNoAutoRequestAck });
+        Messaging::SendFlags sendFlags;
+
+#if !defined(NDEBUG)
```

Comment:
nit: Should we have a separate define for that or you think it's useful to have it always enabled? E.g. BLE endpoint uses `CHIP_BLE_END_POINT_DEBUG_LOGGING_ENABLED`.

Reply:
Currently, I don't think there is a strong reason to introduce a separate module based DEBUG flag for messaging layer. I agree we should consider removing #if !defined(NDEBUG) since ChipLogProgress already have log level.

But for this one, I think we need to align with other CRMP logging to keep it consistent, such as:

#if !defined(NDEBUG)
            ChipLogProgress(ExchangeManager, "Flushed pending ack for MsgId:%08" PRIX32, mPendingPeerAckId);
#endif

Reply:
Yeah, it seems a bit weird that we introduce another implicit log level, but I'm OK with cleaning that later :)